packages/reload-middleware/README.md

```diff
@@ -0,0 +1,86 @@
+#  `@sap-ux/reload-middleware`
+
+The `@sap-ux/reload-middleware` is a [Custom UI5 Server Middleware](https://sap.github.io/ui5-tooling/pages/extensibility/CustomServerMiddleware) for reloading applications automatically during local development. Whenever a file relevant for the SAP Fiori applications is chaged the middleware will refresh the browser tab in which the application is running.
+
+It can be used either with the `ui5 serve` or the `fiori run` commands.
+
+## Configuration Options
+
+| Option       | Type | Description |
+| ------------ | ------------- | ----------- |
+| `path`        | `string \| string[]`      | path or list of paths that should be watched for changes by the reload-middleware. |
+| `[livereload options]`    | `string \| string[] \| number \| boolean`   | Configuration options of [livereload](https://www.npmjs.com/package/livereload#server-api). |
+|`https`| `object`| Configuration for starting the livereload server over https. The middleware supports automatically HTTP/2, when the UI5 server is started with the `--h2` option. In this case it uses by default `$HOME/.ui5/server/server.key` and `$HOME/.ui5/server/server.crt` |
+|`https.key`| `string`| Local filesystem path to the private key for https |
+|`https.cert`| `string`| Local filesystem path to the certificate for https |
+|`[connectOptions]`| `string \| string[] \| number \| boolean`| Configuration options of [connect-livereload](https://www.npmjs.com/package/connect-livereload#options) |
+
+## Configuration Examples
+
+### [Minimal configuration](#minimal-configuration)
+In order to use the middleware this is the minimal configuration that you need to provide in the `ui5.yaml` of your application. By default all files under the `webapp` folder with the following extensions `['html', 'js', 'ts', 'json','xml', 'properties', 'change', 'variant', 'ctrl_variant', 'ctrl_variant_change', 'ctrl_variant_management_change']` will be watched. The `livereload` server is started on port `35729`.
+
```

Comment:
For completenes sake could you please add the snippet
```YAML
server:
  customMiddleware:
  - name: reload-middleware
     afterMiddleware: compression
```

Reply:
Yes, will do

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/1704/commits/612b58a467185e15e56e86c559e3d69423bec118